src/librustc_borrowck/diagnostics.rs

```diff
@@ -263,12 +263,46 @@ fn mutable() {
 You can read more about cell types in the API documentation:
 
 https://doc.rust-lang.org/std/cell/
-"##
+"##,
+
+E0499: r##"
+A variable was borrowed as mutable more than once. Erroneous code example:
+
+```
+let mut i = 0;
+let mut x = &mut i;
+let mut a = &mut i;
+// error: cannot borrow `i` as mutable more than once at a time
+```
+
+Please note that in rust, you can have as many reference on a variable as you
+want, but only one can be mutable. Example:
```

Comment:
You can either have many immutable references, or one mutable reference.

(You cannot have an immutable reference + a mutable reference at any time)

Link to the book on ownership and borrowing for more info